5.3.2 release notes (#268)
hazelcast/management-center-docs: docs/modules/release-notes/pages/5-3-2.adoc

= 5.3.2 Release Notes
:description: These release notes list any new features, enhancements, and fixes that were made between version 5.3.1 and 5.3.2 of Hazelcast Management Center (MC).

{description}

[[enh-532]]
== Enhancements

* **SQL Browser:**

> ** Added an ability to see a query from Query history without execution. [MC-2091]

Reviewer: ```suggestion
** Added the ability to view a query from the query history. [MC-2091]
```
Is this what you mean? Otherwise it reads like you can just type a query in the browser and then it is saved in the query history?

Author: yes, previously if user wanted to see the historical query it was automatically executed again. Now it's possible just view it

Reviewer: So maybe something like this?

Added the ability to view a query from the query history. The query is no longer automatically executed.

Author: agree, updated

> ** Updated texts in Data Explorer. [MC-2224]

Reviewer: ```suggestion
** Updated UI text on the welcome screen of the Data Explorer. [MC-2224]
```

> * Now a banner is displayed when repartitioning is in progress. [MC-2132]

Reviewer: ```suggestion
* A banner is now displayed when repartitioning is in progress. [MC-2132]
```

> * Improved database error messages. [MC-2063]

Reviewer 2: haha we actually made them worse by suppressing them for security :)))

> * Jetty SNI check is disabled by default. [MC-2276]

Reviewer 3: ```suggestion
* Jetty SNI host check is disabled by default. [MC-2276]
```

> * Jetty SNI host check is disabled by default. [MC-2276]

Reviewer: ```suggestion
* Jetty SNI host check is now disabled by default. [MC-2276]
```

[[fixes-532]]
== Fixes

* **SQL Browser:**

> ** Fixed primary key violation on a simultaneous executions of multiple queries. [MC-2050]

Reviewer: ```suggestion
** Fixed a primary key violation that occurred when executing multiple simultaneous queries. [MC-2050]
```

> * Fixed incorrect Jet metrics during the first minute after MC is connected. [MC-1578]

Reviewer: ```suggestion
* Fixed incorrect Jet metrics that were displayed one minute after connecting to MC. [MC-1578]
```

> * Fixed false positive HeathCheck warnings for maps created from a default config. [MC-2226]

Reviewer: ```suggestion
* Fixed false positive HealthCheck warnings for maps created using the default configuration. [MC-2226]
```

> * MC no longer blocks HTTP calls when a member config is not available. [MC-2250] [MC-2259]

Reviewer: ```suggestion
* MC no longer blocks HTTP calls when a member configuration is unavailable. [MC-2250] [MC-2259]
```

* MC no longer suppresses authentication error messages when Local Security Provider is used. [MC-2232]
* MC no longer throws an exception when deleting a misconfigured cluster connection. [MC-2237]

> * MC no longer throws an exception when WAN publisher name is not set. [MC-2249]

Reviewer: ```suggestion
* MC no longer throws an exception when the WAN publisher name is not set. [MC-2249]
```
